Jenkinsfile

```diff
@@ -46,8 +89,15 @@ pipeline {
                         env.testProvider = "nfs"
                     } else if (body.contains("[test yugabytedb]")) {
                         env.testProvider = "yugabytedb"
+                    } else if (changedFiles.contains("ceph/")) {
+                        env.testProvider = "ceph"
+                        env.testArgs = "min-test-matrix"
```

Comment:
Why is `min-test-matrix` the default? Should we not err on the side of fuller tests?

Reply:
I'm just hacking around this, so no particular reasons :).